### PR TITLE
[bitcoin-core] enable extraPort in statefulset, rev version

### PR DESCRIPTION
**Pull request by deantchi**
adding `extraPorts` to bitcoin statefulset.  
it already exists in the service.yaml manifest but not the statefulset.yaml

### CODE REVIEW

File: hirosystems/bitcoin-core/templates/statefulset.yaml [hunk 1]
```diff
@@ -171,6 +171,9 @@ spec:
               containerPort: {{ .Values.containerPorts.rpc }}
             - name: tcp-p2p
               containerPort: {{ .Values.containerPorts.p2p }}
+            { { - if .Values.extraPorts } }
+            { { - include "common.tplvalues.render" (dict "value" .Values.extraPorts "context" $) | nindent 12 } }
+            { { - end } }
```
Comment (CharlieC3) [inline hirosystems/bitcoin-core/templates/statefulset.yaml:176]: Can we make sure the formatting for the braces matches the formatting elsewhere?

Comment (deantchi) [reply hirosystems/bitcoin-core/templates/statefulset.yaml]: not sure why my copy/paste does this.  its fixed now

File: hirosystems/bitcoin-core/values.yaml [hunk 2]
```diff
@@ -146,6 +146,11 @@ containerPorts:
   ##
   # rpc: 38332
   # p2p: 38333
+
+## @param service.extraPorts Extra ports to expose in stacks-blockchain container
```
Comment (CharlieC3) [inline hirosystems/bitcoin-core/values.yaml:150]:
```suggestion
## @param service.extraPorts Extra ports to expose in bitcoin-core container
```

**Review by CharlieC3 — APPROVED** [review]
:shipit:

**Pull request merged**